apps/client/.browserslistrc

```diff
@@ -0,0 +1,16 @@
+# This file is used by:
```

Comment:
Find out what browserslist do
No need to make any changes to this file

Reply:
Added to "things to learn list"